.flowconfig

```diff
@@ -1,6 +1,6 @@
 [ignore]
 ; We fork some components by platform
-.*/*[.]android.js
```

Comment:
I reckon we can make this comment a bit more descriptive:

```
Ignore Android-specific modules from RN because they will confuse Flow when the same module is exported from two files.
```